nginx/conf.d/server/server.account

```diff
@@ -20,14 +15,6 @@ location /stripe/webhook {
     proxy_pass http://accounts:3000;
 }
 
-location /api/stripe/billing {
-    proxy_pass http://dashboard:3000;
```

Comment:
We should deprecate these instead just deleting them.

Do we know if any developers, like Rift, might be using these?

This might be an option https://stackoverflow.com/questions/13884141/convention-for-http-response-header-to-notify-clients-of-deprecated-api#:~:text=Warn%2Dcode%20299%20is%20generic,API%20I%20will%20integrate%20it.

Reply:
If we are confident that these are unused, then we can delete. I just want to make sure we are in the habit and doing that check first.

Reply:
@MSevey Endpoints aren't actually removed. Their implementation was originally placed in the old dashboard, which is why we were routing requests there.
However, @ro-tex [re-implemented them in skynet-accounts](https://github.com/SkynetLabs/skynet-accounts/pull/80) where they belong :)

So these removals here just route the traffic to skynet-accounts, the functionality hasn't changed.